### DISCUSSION TITLE
Final Group Assignment

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/21S-ASIAAM-191A#158] [Assignments]
### Due 6/10

Please use this thread to submit your [final group assignment](https://github.com/albertkun/21S-ASIAAM-191A/blob/main/final_group_assignment.md).

## Submission

- [Select a presentation slot on the final sheet](https://docs.google.com/spreadsheets/d/1B5QTtB3iDFuOXT3z6kv1Dwr1eNku9AeVAHF55YEZXFs/edit#gid=350430430)
- Separately take the [individual group evaluation](https://forms.office.com/r/rVgknaiFUd)
- Post your group's repository and website link below:

ex.
Group Repo: https://github.com/akochaphum/my-awesome-group
Group website: https://akochaphum.github.io/my-awesome-group/index.html

**Comment by lvalles10** [comment]
Reimagining 6th:

Repo: https://github.com/lvalles10/Reimagine-6th
Website: https://lvalles10.github.io/Reimagine-6th/temp%20holding%20place/index.html

**Reply by ghost** [reply]
Updated website: https://lvalles10.github.io/Reimagine-6th/

**Comment by rh5140** [comment]
Tenants' Rights Mapping Project:

Repo: [https://github.com/skarakas1/Tenants-Rights-Mapping-Project](https://github.com/skarakas1/Tenants-Rights-Mapping-Project)
Website: [https://skarakas1.github.io/Tenants-Rights-Mapping-Project/index.html](https://skarakas1.github.io/Tenants-Rights-Mapping-Project/index.html)

**Comment by alizezila** [comment]
Migration Justice: 

Repo: https://github.com/annyrviloria/21S-ASIAAM-191A-GroupProject
Website: https://annyrviloria.github.io/21S-ASIAAM-191A-GroupProject/index.html

**Comment by KenNL42** [comment]
Mental Health:
Repo: https://github.com/KenNL42/ASIAAM-191A-Final-Project/
Website: https://kennl42.github.io/ASIAAM-191A-Final-Project/index.html

**Comment by jason-muljadi** [comment]
SEA Admit Tours:
Repo: https://github.com/jason-muljadi/SEAing-Color
Website: https://jason-muljadi.github.io/SEAing-Color/index.html

**Comment by rachan2023** [comment]
Against Asian Hate
Website: https://gu-ana.github.io/21S-191A-Against-Asian-Hate/Final%20project/
Repo: https://github.com/gu-ana/21S-191A-Against-Asian-Hate/tree/rachan2023-newcode/Final%20project

**Comment by Wny-Duong** [comment]
West LA Food Empowerment Project
SITE:https://wny-duong.github.io/WebDev-GIS-Health-Equity/
REPO:https://github.com/Wny-Duong/WebDev-GIS-Health-Equity
